### PR TITLE
Added PR and bug report templates

### PR DESCRIPTION
**Pull request by mmacai**

### CODE REVIEW

**Comment by mmacai** [comment]
@melissaSchmidt0302 @georgimld I've added simple templates for PR and bug report. Let me know if there is something you would add/change/remove. Its something I would like to re-use also for other repos.

**Comment by melissaSchmidt0302** [comment]
Looks good, maybe I would add in the template who created the report, so you know who to contact for. So a field like `author` or `creator`.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Description
```
Comment (melissaSchmidt0302) [inline .github/pull_request_template.md:8]: consider also adding here a section on the top like in the bug report. So `name`, `about` and `labels`

**Comment by mmacai** [comment]
> Looks good, maybe I would add in the template who created the report, so you know who to contact for. So a field like `author` or `creator`.

Makes sense, on the other hand, as soon as someone creates a report, his/her name will be automatically there as they created it and own the first comment, or?

**Pull request merged**